code/config.py

```diff
@@ -1,120 +1,360 @@
-# DeepSlide
-# Jason Wei, Behnaz Abdollahi, Saeed Hassanpour
+"""
+DeepSlide
+Contains all hyperparameters for the entire repository.
 
-# This file contains all hyperparameters for the entire repository.
+Authors: Jason Wei, Behnaz Abdollahi, Saeed Hassanpour
 
-import utils
-from utils import *
+Last Modified: November 30, 2019 (Joseph DiPalma)
+"""
+
+import argparse
+from pathlib import Path
```

Comment:
Can you explain why this is better than just a string? 

Should this and argparse be imported in utils?

Reply:
To be honest I don't use pathlib by myself but after some research i found that it's recommended for Python 3.6+ programming as it's more object oriented approach. Well, technically no practical difference for now but it maybe good for us to use it since we have set the dependency of py3.6.

Reply:
@jasonwei20 This is the first time I used pathlib.  I used it since it is now the suggested way to deal with paths in Python.  It removes the need for the messy OS module.